homeassistant/components/plex/media_browser.py

```diff
@@ -160,9 +165,13 @@ def item_payload(item):
 
 def library_section_payload(section):
     """Create response payload for a single library section."""
+    media_class = section.TYPE
+    if media_class == "show":
+        media_class = MEDIA_CLASS_TV_SHOW
```

Comment:
@MartinHjelmare @cgtobi do you have objections to changing the `MEDIA_CLASS_TV_SHOW` constant from "tv_show" to "show"? Would avoid checking for special cases here.

Reply:
It's the one media_class where the constant doesn't match what the integration provides natively, and a change to the constant wouldn't materially affect other integrations.

Reply:
You're right, this is an already-solved problem anyway:
https://github.com/home-assistant/core/blob/1934631d1cba0c1fc208d8e2fcab97c32c3fc195/homeassistant/components/plex/media_browser.py#L147-L151